NitrogenMod/Items/ReinforcedSuitsCore.cs

```diff
@@ -52,15 +52,15 @@ protected ReinforcedSuitsCore(string classID, string friendlyName, string descri
 
         public override GameObject GetGameObject()
         {
-            GameObject prefab = CraftData.GetPrefabForTechType(this.BaseType);
+            GameObject prefab = CraftData.GetPrefabForTechType(BaseType);
             var obj = GameObject.Instantiate(prefab);
 
             return obj;
         }
 
         private void SetEquipmentType()
         {
-            CraftDataHandler.SetEquipmentType(this.TechType, this.DiveSuit);
+            CraftDataHandler.SetEquipmentType(TechType, DiveSuit);
```

Comment:
Changes like this (and there are _many_ throughout this pull request) are just code style change which obfuscate exactly what meaningful change you're trying to make.

Whatever your personal feels are about where `using` statements or whether `this` should precede a property or not is irrelevant.
A pull request that is 90% code style change and 10% actual change is unhepful.

I would sincerely recommend either starting over: removing the Assembly-CSharp DLLs and reverting any non-functional changes like in this file.

Reply:
This could be just Visual Studio automatically removing these based on styling settings.

Reply:
It's indeed Jetbrains just auto-applying that style setting.